src/main/resources/data/pieces/1a902141-d8c6-4cf3-8a19-193eca007478.json

```diff
@@ -5,8 +5,8 @@
   "format": "Physical",
   "itemId": "15447c41-bc6a-4600-96a4-a1ce7f44c62a",
   "locationId": "fcd64ce1-6995-48f0-840e-89ffa2288371",
-  "poLineId": "2fe6c2dd-3700-4a53-a624-1159cfd7f8ce",
+  "poLineId": "50fb6ae0-cdf1-11e8-a8d5-f2801f1b9fd1",
+  "receivedDate": "2019-01-11T00:00:00.000Z",
   "receivingStatus": "Expected",
```

Comment:
Better to set `receivingStatus` to `Received`

Reply:
`receivingStatus` updated